install.sh

```diff
@@ -135,6 +135,21 @@ function use_local_users()
   fi
 }
 
+function use_grafana_telegraf()
+{
+  local use_grafana
+  local use_telegraf
+  use_grafana=$(yq eval '.monintoring.grafana' config.yml)
```

Comment:
Typo here, should be monitoring

Reply:
monitoring may not be present

Reply:
fixed the typo. 
when not present, this evaluates to 'null'. action is taken if the value is 'false'